### DISCUSSION TITLE
Intel i9 64G RAM 2 x M.2 1TB Ubunto 21.04 Freezing completely

### DISCUSSION THREAD

**Discussion by laz954** [madMAx43v3r/chia-plotter#474] [Q&A]
I tried running the software:
./build/chia_plot -n 1 -r 14 -u 128 -t /mnt/ssd2/ -2 /mnt/ssd1/ -d /mnt/hdd3/ 

Number of Threads: 14
Number of Buckets: 2^7 (128)
Working Directory:   /mnt/ssd2/
Working Directory 2: /mnt/ssd1/ 
[P1] Table 1 took 15.2488 sec
[P1] Table 2 took 1420011 sec, found 4294911201 matches
At this point the machine just freezes completely including the OS Ubuntu 21.04
M.2's are Sabrent Rocket 4.0 1TB (RKT401.3)Nvme's
Any suggestions on what to look for, I can currently do 3 plots @ 2 Threads each on each Nvme but it takes 6 hrs, that's why I wanted to use the madMAx43v3r plotter software, friend is doing the same but in windows without any issues.
I need help any suggestions ? and or guidance would be greatly appreciated. Tried stressing CPU's checking memory, even blacklisted Video for know issue of Ubuntu freezing, works with standard chia plotting software.

**Comment by seeker003** [comment]
What is your configuration, CPU, RAM, etc.

**Comment by isipisi89** [comment]
is your cpu overclocked? it seems like running unstable for me.

**Comment by laz954** [comment]
Thank you all, it looks like after 20 hrs of testing it's one of the 32G RAM Modules  T-Force VulcanZ DDR4 3200. that's causing the freezing, very erratic, took allot of testing with memtest86. It's working on 32G a little slow but working.
